Aztec/Classes/TextKit/TextStorage.swift

```diff
@@ -171,53 +197,52 @@ open class TextStorage: NSTextStorage {
     /// - Returns: the preprocessed string.
     ///
     fileprivate func preprocessAttachmentsForInsertion(_ attributedString: NSAttributedString) -> NSAttributedString {
-        
+        assert(attachmentsDelegate != nil)
```

Comment:
Maybe out of scope of this PR, but I think we should some kind of default implementation of the attachments Delegate that just work, and then allow users of the library to customize/override it.